src/client/components/ProgressBar.components/ProgressBar.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ProgressBar from './ProgressBar';
+
+export default {
+  title: 'ProgressBar',
+  component: ProgressBar,
+  argTypes: {
+    level: {
+      control: { type: 'range', min: 0, max: 100, step: 1 },
+    },
+    backgroundColor: {
+      control: { type: 'color' },
+    },
+    alphaLevel: {
+      control: { type: 'range', min: 0, max: 1, step: 0.01 },
+    },
+  },
+};
+
+const Template = ({ level, backgroundColor, alphaLevel }) => {
+  return (
+    <ProgressBar
+      level={level}
+      backgroundColor={backgroundColor}
+      alphaLevel={alphaLevel}
+    />
+  );
+};
+
+export const Progress = Template.bind({});
+Progress.args = {
+  level: '20',
+  backgroundColor: 'red',
+  alphaLevel: '0.5',
+};
```

Comment:
You have to add this for the `propTypes` warnings :) 
![image](https://user-images.githubusercontent.com/16811871/115148113-6d890b80-a05e-11eb-9a60-2772c7aa3d07.png)

Reply:
Got that, thanks